aml-development/frontend
# feat(profile): allow editing of category subscriptions

AMLOS-695

**Description**
Allow a user to change their category subscriptions

**Acceptance Criteria**
A user can subscribe to a category for notifications
A user can unsubscribe to category notifications from the user profile page.

**How To Test**
Pull `category_subscription` from `frontend`

Log in as a user
Go to the edit profile page --> preferences
Ensure the Category Subscriptions drop-down has categories to select from
Change the categories and save the profile
Ensure the profile gets saved and data is persisted
